lib/index.js

```diff
@@ -97,17 +110,27 @@ AdRoll.prototype.track = function(track) {
   del(customProps, 'orderId');
   del(customProps, 'id');
   del(customProps, 'sku');
-  if (!is.empty(customProps)) data.adroll_custom_data = customProps;
+  if (!is.empty(customProps)) {
+    each(customProps, function(key, val) {
```

Comment:
Let's use `foldl` here and produce a new object; I'd like to move toward using `each` only to produce side effects so that they'll stand out.